### ISSUE TITLE
Possible to run multiple tests at once?

### ISSUE THREAD

**Issue by maxp-edcast** [judofyr/minitest-line#16]
Say there are two tests, one at line 20 and one at line 40. Is there a way to run both of these with a single command?

**Comment by grosser** [comment]
no, but please add a PR for it, should be a neat little feature ... I think rspec/cucumber support that ... use their syntax as inspiration

**Comment by judofyr** [comment]
I wouldn't be opposed to adding it, but I don't understand see the workflow. If I have multiple tests I want to run I'd rather create a filter. My use case for minitest-line is more "quickly glance at the line number I'm current on" in order to run the current test.

**Comment by maxp-edcast** [comment]
@judofyr I'm not aware of this filter capability, can you give an example? Also what do you mean by quickly glance at the line number? Like when a test fails you can quickly see which one it is and try running it again?

**Comment by judofyr** [comment]
Filtering by test name:

```ruby
ruby test_foo.rb -n /regexp_that_matches_test_name/
```

My use case for minitest-line was mainly while I was writing a test I wanted a quick way to run it. As I write/tweak a test I just need the filename and the approximate line number and I can quickly run it.

**Comment by Nakilon** [comment]
> I don't understand see the workflow

When you look at tests run report and decide to rerun a specific several of them.
